services/Core.ts

```
@@ -206,9 +214,69 @@ export const startCronJobs = () => {
   logger.debug('Multi-chain cron job setup complete.');
 };
 
+export async function recoverStuckFinalizedDeposits(): Promise<void> {
+  logger.info('Recovering stuck finalized deposits...');
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)